### PR TITLE
Fix HUD timer LCD shadow under 1 minute

### PR DESCRIPTION
**Pull request by TheDarkSid3r**

### CODE REVIEW

File: Tweaks/TweaksAssembly/BombStatus.cs [hunk 1]
```diff
@@ -81,7 +81,7 @@ public void LateUpdate()
 		{
 			string formattedTime = currentBomb.GetFullFormattedTime;
 			TimerPrefab.text = formattedTime;
-			TimerShadowPrefab.text = formattedTime.Select(character => character == ':' ? character : '8').Join("");
+			TimerShadowPrefab.text = formattedTime.Select(character => character == ':' || character == '.' ? ':' : '8').Join("");
```
Comment (samfundev) [inline Tweaks/TweaksAssembly/BombStatus.cs:84]: Why did you change it from `character` to `':'`? Shouldn't the shadow text be a period as well?
 
 Also, could you use the `.EqualsAny` method to simplify the logic?

**Review by samfundev — CHANGES_REQUESTED** [review]

File: Tweaks/TweaksAssembly/BombStatus.cs (hunk 1 above)
Comment (TheDarkSid3r) [reply Tweaks/TweaksAssembly/BombStatus.cs:84]: 1. The in-game bomb timer's shadow always displays `:`, I just thought it made sense.
2. Yes, I'm a dumbass.
